ruff.toml

```diff
@@ -0,0 +1,118 @@
+
```

Comment:
Yeah, I had it in the `pyproject.toml`.  I thought I'd try to prevent myself from accidentally committing it with this PR, so I moved it to a separate file. ...And then I proceeded to commit that new file to the PR.  :)